test/unittests/mongoBackend/mongoUpdateContext_test.cpp

```diff
@@ -9199,110 +9199,6 @@ TEST(mongoUpdateContextRequest, servicePathEntityUpdate_3levels)
 
 /* ****************************************************************************
 *
-* servicePathEntityUpdates_2levels -
-*
-*/
-TEST(mongoUpdateContextRequest, servicePathEntityUpdate_2levels)
```

Comment:
prepareDatabaseWithServicePaths creates two entities:
- E1/T1 with service path "/home/kz/01"
- E1/T1 with service path "/home/kz/02"

mongoUpdateContext is called with service path "/home/kz", not "/home/kz/#", but still, both entities are found and updated.

It seems to be this is the exact behaviour we want to avoid (that this PR fixes).
Have I misunderstood something/everything?